src/ImageSharp/ColorProfiles/KnownIlluminants.cs

```diff
@@ -12,60 +12,60 @@ namespace SixLabors.ImageSharp.ColorSpaces;
 /// <br />
 /// Descriptions taken from: http://en.wikipedia.org/wiki/Standard_illuminant
 /// </remarks>
-public static class Illuminants
+public static class KnownIlluminants
 {
     /// <summary>
     /// Incandescent / Tungsten
     /// </summary>
-    public static readonly CieXyz A = new CieXyz(1.09850F, 1F, 0.35585F);
+    public static readonly CieXyz A = new(1.09850F, 1F, 0.35585F);
```

Comment:
Maybe it's better to have this as properties like
```suggestion
    public static CieXyz A => new(1.09850F, 1F, 0.35585F);
```
because (I didn't measure it here, so only theory / feeling):
* with the static readonly it's a memory load (potentially not from cache, but from RAM, so hard to measure in micro-benchmarks)
* with the property it's a simple assignment of three floats
* the JIT sees what's going on, so can potentially optimize the use of the three floats (which isn't that easy w/ loading the struct from memory)

Note: that would be a change in the public API.

Reply:
I've changed the pattern to use the auto property with initializer. That's the pattern we've used elsewhere. 

The types are immutable, and I think this approach should be best for more tightly constrained memory environments.